[2022-08-30T02:25:06.167+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: postgres_operator_dag.create_pet_table scheduled__2020-02-02T00:00:00+00:00 [queued]>
[2022-08-30T02:25:06.181+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: postgres_operator_dag.create_pet_table scheduled__2020-02-02T00:00:00+00:00 [queued]>
[2022-08-30T02:25:06.181+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-08-30T02:25:06.182+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2022-08-30T02:25:06.182+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-08-30T02:25:06.208+0000] {taskinstance.py:1389} INFO - Executing <Task(PostgresOperator): create_pet_table> on 2020-02-02 00:00:00+00:00
[2022-08-30T02:25:06.216+0000] {standard_task_runner.py:52} INFO - Started process 108 to run task
[2022-08-30T02:25:06.219+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'postgres_operator_dag', 'create_pet_table', 'scheduled__2020-02-02T00:00:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/postgres_operator_dag.py', '--cfg-path', '/tmp/tmpfhhgq0gj', '--error-file', '/tmp/tmpwd_2v6ng']
[2022-08-30T02:25:06.220+0000] {standard_task_runner.py:80} INFO - Job 15: Subtask create_pet_table
[2022-08-30T02:25:06.321+0000] {task_command.py:371} INFO - Running <TaskInstance: postgres_operator_dag.create_pet_table scheduled__2020-02-02T00:00:00+00:00 [running]> on host 7f44611b6b02
[2022-08-30T02:25:06.421+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgres_operator_dag
AIRFLOW_CTX_TASK_ID=create_pet_table
AIRFLOW_CTX_EXECUTION_DATE=2020-02-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-02-02T00:00:00+00:00
[2022-08-30T02:25:06.437+0000] {base.py:68} INFO - Using connection ID 'postgres_default' for task execution.
[2022-08-30T02:25:06.442+0000] {sql.py:315} INFO - Running statement: CREATE TABLE IF NOT EXISTS pet (
            pet_id SERIAL PRIMARY KEY,
            name VARCHAR NOT NULL,
            pet_type VARCHAR NOT NULL,
            birth_date DATE NOT NULL,
            OWNER VARCHAR NOT NULL), parameters: None
[2022-08-30T02:25:06.488+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=postgres_operator_dag, task_id=create_pet_table, execution_date=20200202T000000, start_date=20220830T022506, end_date=20220830T022506
[2022-08-30T02:25:06.551+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-30T02:25:06.643+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
